### ISSUE TITLE
Repo: Warning in website builds

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/typescript-eslint#6424]
### Suggestion

The Docusaurus website deployed to https://typescript-eslint.io is working fine, so this issue isn't urgent. But when you `yarn nx run website:build` (`yarn build` from `packages/website`), this warning shows up:

```plaintext
[WARNING] {"moduleIdentifier":"/opt/build/repo/node_modules/babel-loader/lib/index.js??ruleSet[1].rules[5].use[0]!/opt/build/repo/packages/website/src/components/config/utils.ts","moduleName":"./src/components/config/utils.ts","loc":"1:140-145","message":"export 'parse' (imported as 'parse') was not found in 'json5' (possible exports: default)","compilerPath":"server"}
```

...I would prefer we don't have warnings in our builds 😄. It'd be nice if someone could figure out how to fix whatever issue this is complaining about.
